process/smartContract/process.go

```diff
@@ -790,7 +880,7 @@ func (sc *scProcessor) ProcessSmartContractResult(scr *smartContractResult.Smart
 		return process.ErrWrongTypeAssertion
 	}
 
-	storageUpdates, err := sc.argsParser.GetStorageUpdates(scr.Data)
+	storageUpdates, _ := sc.argsParser.GetStorageUpdates(scr.Data)
```

Comment:
No test for the error is needed? If error is not nil, don't we have to do a revert?

Reply:
There is no need to test this, as most of the SCR's will not have valid Data for storageUpdates - they will keep other types of data inside.

Reply:
ok